L1Trigger/L1TTrackerDTC/interface/DTC.h

```diff
@@ -0,0 +1,51 @@
+#ifndef __L1TTrackerDTC_DTC_H__
+#define __L1TTrackerDTC_DTC_H__
+
+#include "L1Trigger/L1TTrackerDTC/plugins/ProducerFwd.h"
+
+namespace L1TTrackerDTC {
+
+  class Settings;
+  class Module;
+  class Stub;
+
+  // representation of an outer tracker DTC board
+  class DTC {
+  private:
+    typedef std::deque<Stub*> Stubs;
```

Comment:
the includes for these STL data formats should be present at the top of the file

Reply:
done